.github/workflows/workflow.yml

```diff
@@ -72,6 +72,7 @@ jobs:
         with:
           ghc-version: ${{ matrix.ghc }}
           cabal-version: ${{ matrix.cabal }}
+          cabal-update: false
```

Comment:
hmm, should not we test the default value (true)? or it is being tested somehow?

Reply:
To test it, we need another toy project here that uses some packages from Hackage.  
(To my surprise, the current test project builds fine without ever having invoked `cabal update`.  I just uses GHC packages.)

Reply:
I now added a test that uses something from Hackage, and it uncovered
- #158